runtime/rococo/Cargo.toml

```diff
@@ -64,7 +64,7 @@ rococo-runtime-constants = { package = "rococo-runtime-constants", path = "./con
 runtime-common = { package = "polkadot-runtime-common", path = "../common", default-features = false }
 primitives = { package = "polkadot-primitives", path = "../../primitives", default-features = false }
 polkadot-parachain = { path = "../../parachain", default-features = false }
-runtime-parachains = { package = "polkadot-runtime-parachains", path = "../parachains", default-features = false }
+runtime-parachains = { package = "polkadot-runtime-parachains", path = "../parachains", default-features = false, features = ["vstaging"] }
```

Comment:
Will this also enable it for every runtime? Usually this is how feature resolution in a workspace works. But I'm not sure given we're using substrate-wasm-builder cc @bkchr

Reply:
Yeah on for the native side, it will be enabled for every runtime

Reply:
That's correct but with the runtime api versioning we no longer need this feature flag (I removed it).